test/linalg/arnoldi.jl

```diff
@@ -31,6 +31,10 @@ using Base.Test
         (d,v) = eigs(a, nev=3)
         @test a*v[:,2] ≈ d[2]*v[:,2]
         @test norm(v) > testtol # eigenvectors cannot be null vectors
+        (d,v) = eigs(a, I, nev=3) # test two argument eigs
+        @test a*v[:,2] ≈ d[2]*v[:,2]
+        @test norm(v) > testtol # eigenvectors cannot be null vectors
+        @test_warn "Use symbols instead of strings for specifying which eigenvalues to compute" eigs(a, which="LM")
```

Comment:
Why is this a thing? Shouldn't we deprecate the change from `String` -> `Symbol` for the `which` kwarg instead of this warning?
Unrelated to this PR though, I can submit a PR for the deprecation if thats the correct thing to do.